aries-site/src/pages/index.js

```diff
@@ -88,7 +83,7 @@ const Index = () => {
       <Meta title={title} description={pageDetails.seoDescription} />
       <Box>
         <Intro>
-          <Box pad={{ horizontal: calcPad(size) }} width={{ max: '900px' }}>
+          <Box width={{ max: '700px' }}>
```

Comment:
Can we add a comment as to how this value was arrived at?

Also, the `xxxsmall` value below looks suspect. I don't believe we have a value for that.